jobs/aws/eks-anywhere-build-tooling/tinkerbell-helm-presubmits.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+presubmits:
+  aws/eks-anywhere-build-tooling:
+  - name: tinkerbell-helm-presubmit
+    always_run: false
+    run_if_changed: "^build/lib/.*|Common.mk|projects/tinkerbell/tinkerbell-helm/.*"
+    cluster: "prow-presubmits-cluster"
+    max_concurrency: 10
+    skip_branches:
```

Comment:
We had this because presubmits running on older release branches, where new projects didn't exist, would fail complaining about the project path not existing. But we don't need it now since we have the project path check. If you can make a sweep and remove this from all prowjobs that would be great, but if you wanna isolate just the new presubmit to this PR that's fine too, we can cleanup later.

Reply:
I can open a separate to clean the others out. Want to keep this one isolated to just the new job